### ISSUE TITLE
diff support

### ISSUE THREAD

**Issue by davidmnoriega** [retr0h/ansible-logrotate#30]
Great module, but looks like when you run it in check/diff mode, the task is skipped. Adding check and diff support would be a great feature.

**Comment by retr0h** [comment]
Yeah, I haven't put any effort into check/diff because I haven't found check mode very helpful.  Check mode with this module alone would probably be fine, but when incorporated into real world playbooks, check mode is pretty worthless.
